### ISSUE TITLE
Kibana / analytics doesn't bootstrap with tk2 cicd

### ISSUE THREAD

**Issue by seathan11** [TykTechnologies/tyk-demo#31]
When trying to bootstrap analytics with tyk2 cicd the kibana server never responds and bootstrap fails.  It works when cicd is not selected.

./up.sh analytics tyk2 cicd

![image](https://user-images.githubusercontent.com/4502637/85793216-aa1bfd00-b702-11ea-9662-85e0e6c13b95.png)

**Comment by davegarvey** [comment]
I suspect the problem here may be that Docker doesn't have enough resources to cope with this deployment.

I tried to replicate the problem and found that I also wasn't able to deploy these deployments successfully. The bootstrap script also got stuck waiting for Kibana to response ok. So I cancelled, brought it down, and increased the resources available to Docker. I gave it access to 6 CPUs and 8GB RAM. Then I tried again and it worked ok.

So, try again, but increase the resources first.

If you still get a problem, check the logs of the Kibana container.
